README.md

```diff
@@ -4,16 +4,20 @@ This repository is supposed to be used as a showcase project to show the usage o
 
 
 # Content
-Below is a tabel of the content fro this project.
+Below is a summary of the content from this project.
 
 ## 1. Shadow DOM
-The `showcase.neodymium.tests.shadowdom` package contains the uscases for the shadow DOM automation.
+The `showcase.neodymium.tests.shadowdom` package contains the use cases for the shadow DOM automation. Please also have a look at our [wiki](https://github.com/Xceptance/neodymium-library/wiki/Shadow-DOM-Testing) for more information.
 #### 1.1 General Shadow DOM
-In `TextTest.java` you can find a simple usecase on how to automatically test shadow DOM elements
+In `TextTest.java` you can find a simple use case on how to automatically test shadow DOM elements.
 #### 1.2. Nested Shadow DOM
-In `CheckboxTest.java` there is are more sophisticated usecases for shadow DOM including nested shadow DOM's.
+`CheckboxTest.java` contains more sophisticated use cases for shadow DOM including nested shadow DOMs. More information can be found [here](https://github.com/Xceptance/neodymium-library/wiki/Shadow-DOM-Testing).
+
 ## 2. Localization
-Under `showcase.neodymium.tests.localization` you can find how to use the localization feature of neodymium. Note that all the translated strings can be found in the `localization.yaml` in the config folder.
+The tests in the `showcase.neodymium.tests.localization` package highlight use the [localization](https://github.com/Xceptance/neodymium-library/wiki/Localization) feature of neodymium is used. Note that all the translated strings and their keys can be found in the `config/localization.yaml` file. 
+## 3. Web slider 
```

Comment:
If I'm seeing this correctly, this isn't part of the current `development` branch. Hence, this section should be removed and added when the web slider actually comes.

Reply:
As far as i know @jseng92 is working on this and added this section to the README. I would leave this in as this is basically in already.

Reply:
It was added by him but we don't have a pull request for the slider yet. Hence, we would add something to the Readme, which isn't here yet and even if there was a pull request, we would introduce a state which does not have the slider-code but tells us in the Readme that their would be. Instead, what is added here, should rather be added to the pull request for the slider.

In other words, even if it is basically in already, it doesn't belong in this merge request and should be removed.

Reply:
Ok, i edited out.